src/uct/ugni/rdma/ugni_rdma_ep.c

```diff
@@ -250,13 +243,14 @@ ucs_status_t uct_ugni_ep_put_bcopy(uct_ep_h tl_ep, uct_pack_callback_t pack_cb,
     uct_ugni_ep_t *ep = ucs_derived_of(tl_ep, uct_ugni_ep_t);
     uct_ugni_rdma_iface_t *iface = ucs_derived_of(tl_ep->iface, uct_ugni_rdma_iface_t);
     uct_ugni_base_desc_t *fma;
+    size_t length;
 
-    UCT_UGNI_ZERO_LENGTH_POST(length);
-    UCT_CHECK_LENGTH(length, iface->config.fma_seg_size, "put_bcopy");
```

Comment:
yes, because now the pack function can override the buffer, so the check after it happened is useless